cmd/internal/find/primary/meta.go

```diff
@@ -70,58 +70,19 @@ const metaDetailedDescription = `
 NOTE: https://puppetlabs.github.io/wash/tutorials/02_find/meta-primary
 contains a hands-on tutorial of the meta primary.
 
-The meta primary constructs a predicate on the entry's metadata. By
```

Comment:
This was all deleted b/c much of this stuff will be in the meta primary tutorial (seems like it makes sense to put it there). Am open to other ideas of restructuring this.

Reply:
Ok, I agree most of that isn't needed as a reference, just the first time you encounter the concept.